backend/app/categories/controller.py

```diff
@@ -18,8 +18,24 @@ async def get_category_route(category_id: str,
                              db: AsyncIOMotorClient = Depends(get_database)) -> ItemsInResponse:
     """Get the items under a given category"""
     # TODO add fetching from category
-    return ItemsInResponse()
+    from ..models.base import CategoryEnum
```

Comment:
Imports should always be at the top level. Read through python good practices before contributing